main.nf

```diff
@@ -105,14 +128,16 @@ include { ORTHOFINDER_PHYLOHOGS                     } from './modules/local/orth
 //
 // MODULE
 //
-// Modules being run twice (for MCL testing and full analysis)
-// needs to be included twice under different names.
-include { BUSCO as BUSCO_SHALLOW                    } from './modules/nf-core-modified/busco'
-include { BUSCO as BUSCO_BROAD                      } from './modules/nf-core-modified/busco'
 include { DIAMOND_BLASTP as DIAMOND_BLASTP_ALL      } from './modules/nf-core-modified/diamond_blastp'
-include { DIAMOND_BLASTP as DIAMOND_BLASTP_TEST     } from './modules/nf-core-modified/diamond_blastp'
-include { IQTREE_PMSF as IQTREE_PMSF_ALL         } from './modules/nf-core-modified/iqtree_pmsf'
-include { IQTREE_PMSF as IQTREE_PMSF_REMAINING      } from './modules/nf-core-modified/iqtree_pmsf'
+
+// Full mode only modules
+if (params.workflow_mode == 'full') {
+    include { BUSCO as BUSCO_SHALLOW                    } from './modules/nf-core-modified/busco'
+    include { BUSCO as BUSCO_BROAD                      } from './modules/nf-core-modified/busco'
+    include { DIAMOND_BLASTP as DIAMOND_BLASTP_TEST     } from './modules/nf-core-modified/diamond_blastp'
+    include { IQTREE_PMSF as IQTREE_PMSF_ALL            } from './modules/nf-core-modified/iqtree_pmsf'
+    include { IQTREE_PMSF as IQTREE_PMSF_REMAINING      } from './modules/nf-core-modified/iqtree_pmsf'
```

Comment:
Honestly we should probably be just using the IQTREE module, rather than the IQTREE_PMSF (unless requested?) - the latter is significantly more involved than IQTREE, which already is much more heavy than FASTTREE